packages-common/nanoflow-commons/src/client/Refresh.ts

```diff
@@ -0,0 +1,32 @@
+// This file was generated by Mendix Studio Pro.
+//
+// WARNING: Only the following code will be retained when actions are regenerated:
+// - the import list
+// - the code between BEGIN USER CODE and END USER CODE
+// - the code between BEGIN EXTRA CODE and END EXTRA CODE
+// Other code you write will be lost the next time you deploy the project.
+
+// BEGIN EXTRA CODE
+// END EXTRA CODE
+
+/**
+ * @param {string} entityObject - Pass mxObject in order to trigger object subscriptions
+ * @param {string} entityName - Pass entityName in order to trigger entity subscriptions
+ * @returns {Promise.<void>}
+ */
+export async function Refresh(entityObject?: mendix.lib.MxObject, entityName?: string) {
+    // BEGIN USER CODE
+
+    if (!entityObject && !entityName) {
+        return Promise.reject(new Error("One of the input parameters are required"));
+    }
+
+    return new Promise(resolve => {
+        mx.data.update({
+            ...(entityObject && { guid: entityObject.getGuid() }),
+            ...(entityName && { entity: entityName }),
```

Comment:
`entityName` and `entityObject`  can never be empty here, they are checked in line 20, you could use the ! makes the code easier on the eyes.

Reply:
@Andries-Smit One of them can be empty and the one that is shouldn't be in there :) It is && not || :)

Reply:
Indeed. sharp!